requirements/prod.in

```diff
@@ -1,4 +1,5 @@
 Django==3.1.7
+autobahn==21.2.1
```

Comment:
Needed to pin this back, because later versions require newer versions of [cryptography](https://github.com/crossbario/autobahn-python/blob/v21.3.1/setup.py#L256), which conflicts with requirements of [cumulusci](https://github.com/SFDO-Tooling/CumulusCI/blob/v3.29.0/requirements.txt#L7).